memory.py

```diff
@@ -1,8 +1,9 @@
 import sys
 import re
-from nparser import Parser
+from parse import Parser
+
+expression = sys.argv[1] if len(sys.argv) > 1 else ""
```

Comment:
??? why we need this condition

Reply:
testing breaks if this condition is not there bc theres no arguments